src/newcharacter.cpp

```diff
@@ -142,12 +142,14 @@ struct points_left {
 
     std::string to_string()
     {
-        if( limit == MULTI_POOL ) {
-            return string_format( _("Points left: %d%c%d%c%d=%d"),
-                stat_points, trait_points >= 0 ? '+' : '-',
-                abs(trait_points), skill_points >= 0 ? '+' : '-',
-                abs(skill_points),
-                stat_points + trait_points + skill_points );
+        if( limit == MULTI_POOL ) {	
```

Comment:
Please remove the trailing white spaces. There are more of them on the other changed lines below. Your editor may have an option to do this automatically.